app/component/SecondaryButton.js

```diff
@@ -6,7 +6,8 @@ import Icon from './Icon';
 
 const secondaryButton = (props, context) =>
   <button
-    className="secondary-button"
+    className={`secondary-button ${props.buttonParams.buttonName} ${props
```

Comment:
Why not use classnames library, as is done in other places? [example](https://github.com/HSLdevcom/digitransit-ui/blob/c358a9f99f217aa129d50650d489281db6cda3e3/app/component/map/Line.js#L50)